ci-operator/step-registry/ipi/conf/gcp/osdisk/disktype-compute/ipi-conf-gcp-osdisk-disktype-compute-commands.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+# check if OCP version will be equal to or greater than the minimum version
+# $1 - the minimum version to be compared with
+# return 0 if OCP version >= the minimum version, otherwise 1
+function version_check() {
+  local -r minimum_version="$1"
+
+  dir=$(mktemp -d)
+  pushd "${dir}"
+
+  cp ${CLUSTER_PROFILE_DIR}/pull-secret pull-secret
+  oc registry login --to pull-secret
+  ocp_version=$(oc adm release info --registry-config pull-secret ${RELEASE_IMAGE_LATEST} --output=json | jq -r '.metadata.version' | cut -d. -f 1,2)
```

Comment:
This is bugged.  RELEASE_IMAGE_LATEST isn't the version that's going to be installed on minor upgrades, it's RELEASE_IMAGE_INITIAL.  This is setting pd-balanced on 4.13->4.14 minor upgrades

Reply:
Sorry for the trouble, we should try to fix the issue soon.